File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.java
import java.util.stream.IntStream;

class Solution {
    public int solution(int[] num_list) {
        int num1 = IntStream.range(0, num_list.length).filter(idx -> idx % 2 == 0).map(idx -> num_list[idx]).sum();
		int num2 = IntStream.range(0, num_list.length).filter(idx -> idx % 2 != 0).map(idx -> num_list[idx]).sum();

		return Math.max(num1, num2);
    }
}

File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.kt
import kotlin.math.max

class Solution {
    fun solution(num_list: IntArray): Int
    = max(num_list.filterIndexed {index, i ->  index % 2 == 0 }.sum(), num_list.filterIndexed {index, i ->  index % 2 != 0 }.sum())
}